lib/graphql_rails/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module GraphqlRails
-  VERSION = '1.2.6'
```

Comment:
I think version bump should be different PR, otherwise, we might have conflicts when each PR comes with its own version